dita/regions/europe/france/unit_a_mock.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE class SYSTEM "../../../dtd/class.dtd">

```

Comment:
IMHO this was a useful reference document during development, but we either don't need it now, or we need to store it somewhere else.  How about putting it in as a comment in the parent issue?

Reply:
could you please tell is it ok to add it in readme.md file

Reply:
removed